FiveCalls/FiveCalls/ScriptReplacements.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ScriptReplacements.swift
+//  FiveCalls
+//
+//  Created by Nick O'Neill on 12/12/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import Foundation
+
+struct ScriptReplacements {
+    static func Replacing(script: String, contact: Contact, location: NewUserLocation?) -> String {
```

Comment:
```suggestion
    static func replacing(script: String, contact: Contact, location: NewUserLocation?) -> String {
```

Reply:
nit: was capitalization intentional?

Reply:
I was using the Go part of my brain to indicate which of these was exported, no reason to do that in swift though 🙃